skrf/media/media.py

```diff
@@ -821,7 +822,9 @@ def line(self, d: NumberLike, unit: str = 'deg',
             z0 = parse_z0(z0)* self.z0
 
         kwargs.update({'z0':z0})
-        result = self.match(nports=2,**kwargs)
+        s_def = kwargs.pop('s_def', S_DEF_DEFAULT)
+        # Zref = Z0 so these S-parameters correspond to traveling waves.
+        result = self.match(nports=2, s_def='pseudo', **kwargs)
```

Comment:
I added this because `NISTMultilineTRLTest2` test failed without it. I investigated it a little bit more and it shouldn't be in theory necessary. The issue is that with the default `power` connecting to thru in embed causes numerical issues. The reason is that when connecting `power` networks they need to be converted to either other two definitions first. The conversion is done by converting to Z-parameters and back. Unfortunately thru Z-parameters have infinities which cause the calculation to be inaccurate when there are both huge and small numbers. I think before this can be merged s_def conversion to and from power should be done in better way.

Reply:
It actually needs to not be power, because it doesn't match with the traveling wave definition since it has the conjugate in reflection coefficient definition.